api/helpers.go

```diff
@@ -115,7 +116,15 @@ func encodeEVMResultsArgs(electionId common.Hash,
 		{Type: address},
 		{Type: uint256SliceNested},
 	}
-	abiEncodedResultsBytes, err := args.Pack(electionId, organizationId, censusRoot, sourceContractAddress, results)
+	// change results from *types.BigInt to *bigInt as args.Pack requires math/big.Int type
+	resultsStd := make([][]*big.Int, len(results))
+	for i, r := range results {
+		resultsStd[i] = make([]*big.Int, len(r))
+		for j, v := range r {
+			resultsStd[i][j] = v.ToStdBigInt()
+		}
+	}
+	abiEncodedResultsBytes, err := args.Pack(electionId, organizationId, censusRoot, sourceContractAddress, resultsStd)
```

Comment:
@mvdan is this loop required for transforming types.BigInt to big.Int? Isn't a way to make a direct casting? (just wondering)

Reply:
unfortunately yes. even though the types are the same in memory, they are different named types in the Go type system, so you can't go from one slice to another directly. we could probably avoid this awkwardness with either generics or avoiding the need for `types.BigInt` to exist at all, but probably for another PR. perhaps raise an issue and I can look at it when we're done with sqlite :)